### PR TITLE
Cors fixes

### PR DESCRIPTION
**Pull request by smallhive**

### CODE REVIEW

**Review by tatiana-nspcc — APPROVED** [review]
Maybe this PR relates to #180, as it's already been closed. And the commit message can look like: "handlers: Fix CORS", but it's up to you. Otherwise, OK.

File: handlers/auth.go [hunk 1]
```diff
@@ -127,8 +127,9 @@ func (a *RestAPI) FormBinaryBearer(ctx echo.Context, params apiserver.FormBinary
 		return ctx.JSON(http.StatusBadRequest, resp)
 	}
 
-	resp := &apiserver.BinaryBearer{
-		Token: base64.StdEncoding.EncodeToString(btoken.Marshal()),
+	resp := &apiserver.BinaryBearer{}
+	if btoken != nil {
```
Comment (roman-khimov) [inline handlers/auth.go:131]: Shouldn't we return some error otherwise?

Comment (smallhive) [reply handlers/auth.go:131]: It's a question. We may return something 400, Empty token

**Comment by smallhive** [comment]
> Maybe this PR relates to #180, as it's already been closed. And the commit message can look like: "handlers: Fix CORS", but it's up to you. Otherwise, OK.

You are right, updated

File: handlers/auth.go (hunk 1 above)
Comment (smallhive) [reply handlers/auth.go]: Added

Comment (roman-khimov) [reply handlers/auth.go]: I think so, likely spec needs to be updated though.

File: handlers/objects.go [hunk 2]
```diff
@@ -480,6 +483,7 @@ func (a *RestAPI) HeadContainerObject(ctx echo.Context, containerID apiserver.Co
 		return ctx.JSON(http.StatusBadRequest, resp)
 	}
 
+	ctx.Response().Header().Set(accessControlAllowOriginHeader, "*")
```
[the remark this answers is not in the document]
Comment (smallhive) [reply handlers/objects.go:486]: I don't see any problem with having this header in the error responses, but I don't mind to move this inside `headByAddress` func

**Review by roman-khimov — APPROVED** [review]

**Pull request merged**